src/docs/metrics/bitmex-perpetual-contract-basis/index.md

```diff
@@ -0,0 +1,71 @@
+---
+title: BitMEX Perpetual Contract Basis
+author: Santiment Team
+date: 2020-06-03
+description: Contract price - index price.
+---
+
+## Description
+
+This metric shows the positive or negative difference between BitMEX perpetual contract’s price and BitMEX index price for a given asset.
```

Comment:
I think it's goo to mention that the difference is in % expressed as a number between -1.0 and 1.0

Reply:
Or is it? I see the measuring unit is dollars, so I guess that will be enough

Reply:
yes, the measuring unit is dollars.